Utilities/bootstrap

```diff
@@ -765,7 +766,7 @@ def main():
                                         "lib", "swift", "pm")
         mkdir_p(bin_install_path)
         mkdir_p(lib_install_path)
-        for product_path in [swift_package_path, swift_build_path, swift_test_path]:
+        for product_path in [swift_package_path, swift_build_path, swift_test_path, swiftpm_xctest_helper_path]:
```

Comment:
I would rather we install this to `<prefix>/libexec/swift/pm/swiftpm-xctest-helper` which is the right place for an executable tool which is only an "implementation detail" of the actual user facing tools.